src/main/resources/application-local.properties

```diff
@@ -1 +1,3 @@
-feature.googleMapsEnabled=false
\ No newline at end of file
+feature.googleMaps.enabled=false
+spring.data.redis.host=localhost
```

Comment:
Masz już in-memory cache, może warto dać feature-flaga? Żeby lokalnie nie trzeba było zaciągać integracji z dockera, ale kto będzie chciał ten sobie włączy możliwość